.eleventy.js

```diff
@@ -177,18 +177,15 @@ module.exports = function (eleventyConfig) {
     if (lang != 'en ' && lang != 'fr') {
       lang = 'en';
     }
-    const content = markdownLibrary.render(children);
 
     return `
-    <div class="code-showcase mb-400">
-      <div class="showcase mb-400 p-400" id="${id}" aria-hidden="true">
-        ${content}
-      </div>
-      <div>
-        <gcds-button button-type="button" button-role="secondary" aria-label="${langStrings[lang].view} - ${name}" onclick="toggleCodeShowcase(this, '${id}');" aria-controls="${id}" aria-expanded="false">${langStrings[lang].view}</gcds-button>
-        <gcds-button button-type="button" button-role="secondary" onclick="copyCodeShowcase(this, '${id}', '${lang}');" onblur="this.innerText = '${langStrings[lang].copy}'">${langStrings[lang].copy}</gcds-button>
-      </div>
-    </div>
+        <div class="code-showcase mb-400">
+          <textarea class="showcase text-light mb-400 p-400" id="${id}" rows="8" aria-hidden="true">${content}</textarea>
```

Comment:
Clever use of a textarea! Would we be able to add a label to avoid it being flagged as an accessibility issue and add a `read-only` attribute to avoid people modifying it.

Reply:
Oops yes, I meant to add that in and then got distracted and forgot.